### PR TITLE
[hi] Add Hindi glossary entry for "master"

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
Add Hindi glossary entry for "master"

### Description

#Fixes Localizes content/en/docs/reference/glossary/master.md to Hindi content/hi/docs/reference/glossary/master.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/master.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  पुराना शब्द, जिसका उपयोग कंट्रोल प्लेन चलाने वाले नोड्स के पर्याय के रूप में किया जाता था।
+
+aka:
+tags:
+- fundamental
+---
+  पुराना शब्द, जिसका उपयोग {{< glossary_tooltip text="control plane" term_id="control-plane" >}} चलाने वाले {{< glossary_tooltip text="nodes" term_id="node" >}} के पर्याय के रूप में किया जाता था।
+ 
+<!--more-->
+मास्टर अभी भी कुछ प्रोविजनिंग टूल्स, जैसे{{< glossary_tooltip text="kubeadm" term_id="kubeadm" >}},और मैनेज्ड सर्विसेज़ द्वारा इस्तेमाल किया जा रहा है, ताकि kubernetes.io/role का उपयोग करके {{< glossary_tooltip text="nodes" term_id="node" >}} को {{< glossary_tooltip text="label" term_id="label" >}} किया जा सके और {{< glossary_tooltip text="control plane" term_id="control-plane" >}} से {{< glossary_tooltip text="pods" term_id="pod" >}} की प्लेसमेंट को नियंत्रित किया जा सके।
```
Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/master.md:15]:
```suggestion
मास्टर शब्द अभी भी कुछ प्रोविजनिंग टूल्स, जैसे{{< glossary_tooltip text="kubeadm" term_id="kubeadm" >}},और मैनेज्ड सर्विसेज़ द्वारा इस्तेमाल किया जा रहा है, ताकि kubernetes.io/role का उपयोग करके {{< glossary_tooltip text="nodes" term_id="node" >}} को {{< glossary_tooltip text="label" term_id="label" >}} किया जा सके और {{< glossary_tooltip text="control plane" term_id="control-plane" >}} से {{< glossary_tooltip text="pods" term_id="pod" >}} की प्लेसमेंट को नियंत्रित किया जा सके।
```

File: content/hi/docs/reference/glossary/master.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: मास्टर (Master)
```
Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/master.md:2]: You can Master in aka

File: content/hi/docs/reference/glossary/master.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+id: master
+date: 2020-04-16
+short_description: >
+  पुराना शब्द, जिसका उपयोग कंट्रोल प्लेन चलाने वाले नोड्स के पर्याय के रूप में किया जाता था।
+
+aka:
+tags:
+- fundamental
+---
+  पुराना शब्द, जिसका उपयोग {{< glossary_tooltip text="control plane" term_id="control-plane" >}} चलाने वाले {{< glossary_tooltip text="nodes" term_id="node" >}} के पर्याय के रूप में किया जाता था।
```
Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/master.md:12]:
```suggestion
{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} 
```

Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/master.md:12]:
```suggestion
  पुराना शब्द, जिसका उपयोग {{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} चलाने वाले {{< glossary_tooltip text="नोड्स" term_id="node" >}} के पर्याय के रूप में किया जाता था।
```

File: content/hi/docs/reference/glossary/master.md (hunk 1 above)
Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/master.md:15]:
```suggestion
मास्टर अभी भी कुछ प्रोविजनिंग टूल्स, जैसे{{< glossary_tooltip text="kubeadm" term_id="kubeadm" >}},और मैनेज्ड सर्विसेज़ द्वारा इस्तेमाल किया जा रहा है, ताकि kubernetes.io/role का उपयोग करके {{< glossary_tooltip text="नोड्स" term_id="node" >}} को {{< glossary_tooltip text="लेबल" term_id="label" >}} किया जा सके और {{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} से {{< glossary_tooltip text="पॉड्स" term_id="pod" >}} की प्लेसमेंट को नियंत्रित किया जा सके।
```

Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/master.md]: @utkarsh-singh1 Updated as per your suggestions! pls review back!

File: content/hi/docs/reference/glossary/master.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+id: master
+date: 2020-04-16
+short_description: >
+  पुराना शब्द, जिसका उपयोग कंट्रोल प्लेन चलाने वाले नोड्स के पर्याय के रूप में किया जाता था।
+
+aka:
+tags:
+- fundamental
+---
+  पुराना शब्द, जिसका उपयोग {{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane" >}} चलाने वाले {{< glossary_tooltip text="नोड्स" term_id="node" >}} के पर्याय के रूप में किया जाता था।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/master.md:12]: Use that predefined term  of glossary tooltips in hindi whose translation is already present in our glossary else use default language en

**Review by niranjandarshann — COMMENTED** [review]
Use the same Pattern everywhere into the docs.

File: content/hi/docs/reference/glossary/master.md (hunk 4 above)
Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/master.md:12]: Sure, got it.

**Comment by CloudShiva7cr** [comment]
@niranjandarshann @utkarsh-singh1 Can you review once, have made necessary changes as per your suggestions,  Thanks!!

**Comment by CloudShiva7cr** [comment]
@divya-mohan0209 Can you review once and let me know if any more changes needed!! Thanks

**Pull request closed without merging**